.travis.yml

```diff
@@ -145,7 +145,6 @@ jobs:
       env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
 
     - name: PHP and JavaScript unit tests (7.4, WordPress trunk, with code coverage)
-      if: branch = develop AND type = push
```

Comment:
I think this was put in place because otherwise the PR builds take very long, no?

Reply:
Yea that makes sense, would be great to have coverage for each PR nonetheless.